src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
+    private static final Random random = new Random();
+    private static ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
let's remove a static modifier as it should be a regular field. Also make your code consistent and add final to the colorSupplier